### PR TITLE
🟢 Centralize bert CPU e2e aml tests

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes
1. Centralize bert CPU e2e aml tests to reduce confusion.
2. Re-order the ci job by model type.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by jambayk — DISMISSED** [review]
Thanks for cleaning up the test!

File: .azure_pipelines/olive-ci.yaml [hunk 1]
```diff
@@ -39,21 +39,21 @@ jobs:
...
-      resnet_vitis_ai_ptq_cpu:
-        exampleFolder: resnet
-        exampleName: resnet_vitis_ai_ptq_cpu
       bert_ptq_cpu_aml:
         exampleFolder: bert
         exampleName: bert_ptq_cpu_aml
       bert_ptq_cpu_docker:
         exampleFolder: bert
         exampleName: bert_ptq_cpu_docker
+      resnet_ptq_cpu:
```
Comment (guotuofeng) [inline .azure_pipelines/olive-ci.yaml:48]: It seems the file is not changed. right?

Comment (jambayk) [reply .azure_pipelines/olive-ci.yaml:48]: Yeah, looks like the tests were reordered in alphabetical order. But `cifar..` test is before `bert..`. Maybe @trajepl can update that too?

Comment (trajepl) [reply .azure_pipelines/olive-ci.yaml:48]: Updated.

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**